src/util/server.ml

```diff
@@ -129,14 +126,14 @@ let virtual_changes file =
 let increment_data (s: t) file reparsed = match !Serialize.server_solver_data with
   | Some solver_data when reparsed ->
     let changes = CompareCIL.compareCilFiles s.file file in
-    let old_data = Some { Analyses.cil_file = s.file; solver_data } in
+    let old_data = Some { Analyses.solver_data } in
```

Comment:
This seems a bit weird... Why make a record that has only one member?

Reply:
I didn't want to introduce additional refactoring on top of this. Depending on how #357 and #545 are done, the record might gain additional fields again or could be removed entirely. Didn't want to refactor something that is bound to be refactored more extensively anyway.